api/internal/plugins/builtinconfig/transformerconfig.go

```diff
@@ -7,6 +7,7 @@ import (
 	"log"
 	"sort"
 
+	"github.com/pkg/errors"
```

Comment:
`"github.com/pkg/errors"` looks like an already [unmaintained and archived](https://github.com/pkg/errors/issues/245).

Could you think of using `fmt.Errorf()` instead of `errors.Wrap()` if you don't have any reason?

Reply:
> `"github.com/pkg/errors"` looks like an already [unmaintained and archived](https://github.com/pkg/errors/issues/245).
> 
> Could you think of using `fmt.Errorf()` instead of `errors.Wrap()` if you don't have any reason?

done! thanks for making me aware of that